src/docs/product/issues/performance-issues/n-one-queries.mdx

```diff
@@ -8,13 +8,23 @@ description: "Learn more about N+1 queries and how to diagnose and fix them."
 
 _N+1 queries_ are a performance problem in which the application makes database queries in a loop, instead of making a single query that returns all the information at once. Each database connection takes some amount of time, so querying the database in a loop can be many times slower than doing it just once. This problem often occurs when you use an object-relational mapping (ORM) tool in web frameworks like Django or Ruby on Rails.
 
+## Detection Criteria
+
+This detector looks for a set of sequential, non-overlapping database spans with similar descriptions. It also uses a few extra criteria:
```

Comment:
Using "this" here tends to connote that you've talked about the detector before. 

```suggestion
The detector for performance issues looks for a set of sequential, non-overlapping database spans with similar descriptions. It also uses the following criteria:
```